src/core/Codend.Application/Sprints/Commands/UpdateSprint/UpdateSprintValidator.cs

```diff
@@ -0,0 +1,46 @@
+using Codend.Application.Extensions;
+using Codend.Domain.ValueObjects;
+using FluentValidation;
+using static Codend.Application.Core.Errors.ValidationErrors.Common;
+
+namespace Codend.Application.Sprints.Commands.UpdateSprint;
+
+/// <summary>
+/// Validates <see cref="UpdateSprintCommand"/>.
+/// </summary>
+public class UpdateSprintValidator : AbstractValidator<UpdateSprintCommand>
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="UpdateSprintValidator"/> class.
+    /// </summary>
+    public UpdateSprintValidator()
+    {
+        RuleFor(x => x.SprintId)
+            .NotEmpty()
+            .WithError(new PropertyNullOrEmpty(nameof(UpdateSprintCommand.SprintId)));
+        
+        When(x => x.Name is not null, () =>
+        {
+            RuleFor(x => x.Name)
+                .NotEmpty()
+                .WithError(new PropertyNullOrEmpty(nameof(UpdateSprintCommand.Name)))
+                .MaximumLength(SprintName.MaxLength)
+                .WithError(new StringPropertyTooLong(nameof(UpdateSprintCommand.Name), StoryName.MaxLength));
+        });
+        
+        RuleFor(x => x.StartDate)
```

Comment:
I think there should be rule checking if enddate is not after startdate.
Edit: or we can leave it as it is if you think period value object own validation is enough and nothing will be infected after.